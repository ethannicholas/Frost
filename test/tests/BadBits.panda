class BadBits {
    method badIf() {
        if 5 {
        }
    }

    method badWhile() {
        while "hello" {
        }
    }

    method badDo() {
        do {
        }
        while self
    }
}