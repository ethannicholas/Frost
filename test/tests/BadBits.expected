BadBits.frost:3:12: error: expected 'frost.core.Bit', but found 'frost.core.Int'
BadBits.frost:8:15: error: expected 'frost.core.Bit', but found 'frost.core.String'
BadBits.frost:15:15: error: expected 'frost.core.Bit', but found 'anonymous.BadBits'
3 errors
